### DISCUSSION TITLE
Diamond template did not support multi-language (i18n)

### DISCUSSION THREAD

**Discussion by pctimhk** [primefaces/community#1035] [PrimeNG Templates] [answered]
I purchased PrimeNg Diamond template but I found that the template did not support multi-language localization (e.g. i18n). Please add it if possible.

**Comment by cagataycivici** [comment, marked as answer]
Hi, translation is out of scope of the templates because we leave it to the end user to implement it the way they prefer. The template mainly provides a topbar, menu and footer. The rest are sample demos and there is no important requirement to add i18n since there are not many built-in texts to translate. In addition, over the course of 10 years of offering templates, we haven't received a demand on translation so not on our roadmap.
